ui/admin/tests/acceptance/host-catalogs/list-test.js

```diff
@@ -146,6 +169,65 @@ module('Acceptance | host-catalogs | list', function (hooks) {
 
     await click(`[href="${urls.hostCatalogs}"]`);
 
-    assert.dom(`[href="${urls.hostCatalog}"]`).exists();
+    assert.dom(`[href="${urls.staticHostCatalog}"]`).exists();
+  });
+
+  test('user can search for a specific host catalog by id', async function (assert) {
+    await visit(urls.projectScope);
+
+    await click(`[href="${urls.hostCatalogs}"]`);
+    assert.dom(`[href="${urls.staticHostCatalog}"]`).exists();
+    assert.dom(`[href="${urls.awsHostCatalog}"]`).exists();
+    assert.dom(`[href="${urls.azureHostCatalog}"]`).exists();
+
+    await fillIn(SEARCH_INPUT_SELECTOR, instances.staticHostCatalog.id);
+    await waitFor(`[href="${urls.awsHostCatalog}"]`, { count: 0 });
```

Comment:
@lisbet-alvarez @cameronperera

I've been using `waitFor` instead of `waitUntil` like this because I found it a bit cleaner, any thoughts?

Reply:
yeah it is cleaner this way and removes the need to use findAll as well! Going forward ill use this instead.